confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -775,6 +777,65 @@ public DataSet showRegion(GetRegionInfoListPlan getRegionsinfoReq) {
     }
   }
 
+  @Override
+  public DataSet showDataNodes(GetRegionInfoListPlan getRegionsinfoReq) {
```

Comment:
The GetRegionInfoListPlan should be generated inside the showDataNodes. It looks weird as a input parameter.

Reply:
fixed~